File: problemas/1000/1047.py
(horaInicio, minutoInicio, horaFim, minutoFim) = map(int, input().split(' '))
minutoInicial = 60 * horaInicio + minutoInicio
minutoFinal = 60 * horaFim + minutoFim

duracao = minutoFinal - minutoInicial if minutoFinal > minutoInicial\
  else 1440 - minutoInicial + minutoFinal

horas = duracao // 60
minutos = duracao % 60

print(f'O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)')

File: problemas/1000/1047.js
const [horaInicio, minutoInicio, horaFim, minutoFim] = require('fs')
.readFileSync('/dev/stdin', 'utf-8')
.split(' ')
.map(x => Number(x));

const minutoInicial = 60 * horaInicio + minutoInicio;
const minutoFinal = 60 * horaFim + minutoFim;

const duracao = minutoFinal > minutoInicial ? minutoFinal - minutoInicial
  : 1440 - minutoInicial + minutoFinal;

const horas = Math.floor(duracao / 60);
const minutos = duracao % 60;

console.log(`O JOGO DUROU ${horas} HORA(S) E ${minutos} MINUTO(S)`);